packages/noya-state/src/selectors/pointSelectors.ts

```diff
@@ -7,6 +8,20 @@ import { visit } from '../layers';
 import { getCurrentPage } from './pageSelectors';
 import { getBoundingRectMap } from './selectors';
 
+export const POINT_RADIUS = 4;
+
+export const isPointInRange = (point: Point, rawPoint: Point): boolean => {
+  if (
```

Comment:
We can simplify by using `return` directly instead of `if/else` here: `return (point.x >= ......)`